src/actions/commands/put.ts

```diff
@@ -417,27 +421,29 @@ class PutCommandVisual extends BaseCommand {
       if (replaceRegisterName === deletedRegisterName) {
         Register.putByKey(deletedRegister.text, deletedRegisterName, deletedRegister.registerMode);
       }
-      return;
```

Comment:
stylewise, I am not 100% sure to remove return here, but I am quite sure code for the alternative case should be in else {...}.

Reply:
Yep seems reasonable to me